docs/conf.py

```diff
@@ -49,7 +51,10 @@
 
 # General information about the project.
 project = 'Scrapy'
-copyright = f'2008–{datetime.now().year}, Scrapy developers'
+build_date = datetime.utcfromtimestamp(
+    int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))
+)
+copyright = f'2008–{build_date.year}, Scrapy developers'
```

Comment:
```suggestion
copyright = f'2008–2022, Scrapy developers'
```

... plus removing the `datetime`, `os`, and `time` imports from this configuration module.

(rationale: it's nice to automate where possible -- and _authorship_ doesn't expire -- but copyright does, and figuring that out (and updating notices like this) usually benefits from human thought and discussion)

Reply:
Although ultimately I decided that it wasn't worth keeping open as a feature request, I'll mention for completeness/connection that I filed a feature request at sphinx-contrib/sphinx-lint#53 to implement a lint checking rule to test for copyright information in Sphinx configuration files.

Something like that, I reasoned initially, would allow continuous integration to check for the presence of outdated and static copyright lines written by human authors.